js/defaults_dialog.js

```diff
@@ -227,15 +239,83 @@ helper.defaultsDialog = (function() {
                 },
                 {
                     key: "roll_rate",
-                    value: 20
+                    value: 18
                 },
                 {
                     key: "pitch_rate",
-                    value: 15
+                    value: 9
                 },
                 {
                     key: "yaw_rate",
-                    value: 9
+                    value: 3
+                },
+		{ 
+		    key: "nav_fw_pos_z_p",
+		    value: 20
+		},
+		{ 
+		    key: "nav_fw_pos_z_d",
+		    value: 5
+		},
+		{ 
+		    key: "nav_fw_pos_xy_p",
+		    value: 50
+		},
+		{ 
+		    key: "fw_turn_assist_pitch_gain",
+		    value: 0.5
+		},
+		{ 
+		    key: "max_angle_inclination_rll",
+		    value: 350
+		},
+		{ 
+                    key: "nav_fw_bank_angle",
+                    value: 35
+		},
+		{ 
+                    key: "fw_p_pitch",
+                    value: 15
+		},
+		{ 
+                    key: "fw_i_pitch",
+                    value: 10
+		},
+		{ 
+                    key: "fw_ff_pitch",
+                    value: 60
+		},
+		{ 
+                    key: "fw_p_roll",
+                    value: 10
+		},
+		{ 
+                    key: "fw_i_roll",
+                    value: 8
+		},
+		{ 
+                    key: "fw_ff_roll",
+                    value: 40
+		},
+		{ 
+                    key: "fw_p_yaw",
+                    value: 20
+		},
+		{ 
+                    key: "fw_i_yaw",
+                    value: 0
```

Comment:
I don't agree with disabling I-term stabilization on yaw as a solution to the rudder counteracting turns. https://github.com/iNavFlight/inav/pull/6387 is a better solution to that so I propose adding `fw_yaw_iterm_freeze_bank_angle = 10` as a default instead.

Reply:
@avsaase this is only for the initial preset. It will be tuned with autotune later. and it is hard to determine a good I term as the rudder size and authority can massively differ from plane to plane. I wanna avoid a yaw spin or stall due to excessive rudder deflection after a bank turn on planes like a tundra or Extra.

Reply:
But Yaw I helps with trimming the plane, which you typically do before autotune.

Reply:
@avsaase hmm actually a good point. maybe a value of 5 is okay as a starting point as yaw trim is usually only minimal as ti can be visually aligned well.